content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -59,3 +51,26 @@ The video snippets below demonstrate these new wizards.
 <video width="99%" controls>
   <source src="/images/release-notes/lwm2m-bulk-reg.mp4" type="video/mp4" />
 </video>
+
+#### LWM2M 1.1 support
+
+We are now supporting the following LWM2M 1.1 features:
+
+- LWM2M Send
+- Composite Operations (read, write, observe, cancel observation)
+- New LWM2M serialization formats: CBOR, SenML JSON, SenML CBOR
+- New LWM2M data formats: Core Link, Unsigned Integers
+
+For more information on our 1.1 support, see our [LwM2M protocol integration guide](/guides/protocol-integration/lwm2m/).
+
+#### LWM2M time timestamp conversion
+
+If a timestamp (resources 5518 and 6050, resource 5 for object 6) is received together with resources of the same object, this timestamp can be used as creation timestamp for measurements, events and alarms. See [Advanced settings](/guides/protocol-integration/lwm2m/#advanced-settings) for details.
```

Comment:
Maybe you can add it needs to be enabled.
See [Advanced settings](...) for details and to enable during device registration.